src/Platform/Microsoft.Testing.Platform/OutputDevice/Terminal/NonAnsiTerminal.cs

```diff
@@ -202,6 +202,9 @@ public void RenderProgress(TestProgressState?[] progress)
                 ResetColor();
                 Append(']');
 
+                Append(' ');
```

Comment:
Let's try to be picky on our PR and to avoid mixing changes together.

Reply:
Totally. https://github.com/microsoft/testfx/pull/3551